src/core.js

```diff
@@ -755,11 +755,30 @@ define([
             }
 
             if (window.analytics) {
-                window.analytics.usage(
-                    "Question Reference",
-                    "Drag and Drop",
-                    _this.data.core.currentlyEditedProperty
-                );
+                var targetType;
+                switch (target[0].id) {
+                    case 'property-relevantAttr':
+                        targetType = "Display";
+                        break;
+                    case 'property-constraintAttr':
+                        targetType = "Validation";
+                        break;
+                    case 'property-calculateAttr':
+                        targetType = "Calculation";
+                        break;
+                    default:
+                        targetType = "Expression Editor";
+                        break;
+                }
+                if (path.contains('casedb')) {
```

Comment:
> Isn't this function also called on drag/drop from the question tree?
 
Yeah. I meant that checking for only single quotes was sufficient because the data browser will only drop single quotes. not double